extras/hook-scripts/add-brick/post/disabled-quota-root-xattr-heal.sh

```diff
@@ -106,7 +106,7 @@ ENABLED_STATE_1="${GLUSTERD_WORKDIR}/hooks/${VERSION}/${VOLUME_OP}/"
 ENABLED_STATE_2="post/${ENABLED_NAME_PREFIX}${VOL_NAME}-${ENABLED_NAME}"
 ENABLED_STATE="${ENABLED_STATE_1}${ENABLED_STATE_2}"
 
-if [ "${THIS_SCRIPT}" != *"${VOL_NAME}"* ]; then
+if [[ "${THIS_SCRIPT}" != *"${VOL_NAME}"* ]]; then
```

Comment:
`[[` is a bash enhancement not supported in standard sh. If we change this we should also change the shebang to `#!/bin/bash` at the beginning of the script.

Reply:
Ok, I've modified.

Reply:
> `[[` is a bash enhancement not supported in standard sh. If we change this we should also change the shebang to `#!/bin/bash` at the beginning of the script.

Ok, I've modified.